index.html

```diff
@@ -10,234 +10,252 @@
-    <div class="menuSection">
-      <h1 class="introduction">I’m Mariane Glad to see you!</h1>
-      <p class="Whoiam">
-        I’m a software developer! I can help you build a product , feature or
-        website Look through some of my work and experience! If you like what
-        you see and have a project you need coded, don’t hestiate to contact
-        me.
-      </p>
-    </div>
-    <div class="menuSection">
-      <a href="#" target="_blank">LET'S CONNECT</a>
-      <ul class="socialMedia">
-        <li>
-          <img src="image/Twitter.png" alt="" />
+  <div class="container">
+    <header>
+      <nav class="navmenu">
+        <h3>My Logo</h3>
+      </nav>
+      <img class="onion" src="image/Union.png" alt="" />
+      <div class="rest">
+        <div>Portfolio</div>
+        <div>About</div>
+        <div>Contact</div>
+      </div>
+    </header>
+    <section class="backgroundimg">
+      <div class="menuSection menuSection1">
+        <h2 class="introduction">I’m Mariane Glad to see you!</h2>
+        <p class="Whoiam">
+          I’m a software developer! I can help you build a product , feature
+          or website Look through some of my work and experience! If you like
+          what you see and have a project you need coded, don’t hestiate to
+          contact me.
+        </p>
+      </div>
+      <div class="menuSection">
+        <a href="#" target="_blank">LET'S CONNECT</a>
+        <ul class="socialMedia">
+          <li>
+            <img src="image/Twitter.png" alt="" />
+          </li>
+          <li>
+            <img src="image/Linkedin.png" alt="" />
+          </li>
+          <li>
+            <img src="image/Meduim.png" alt="" />
+          </li>
+          <li>
+            <img src="image/Github.png" alt="" />
+          </li>
+          <li>
+            <img src="image/angellist.png" alt="" />
+          </li>
+        </ul>
+      </div>
+    </section>
+    <div id="cards">
+      <div id="flex">
+        <div class="card">
+          <img class="img" src="image/card-1.svg" alt="" />
+          <div class="content">
+            <div class="topic">Tonic</div>
+            <div class="sub-topic">
+              <span>CANOPY</span>
+              <span class="dot"></span>
 
-        </li>
-        <li>
-          <img src="image/Linkedin.png" alt="" />
+              <span>Back End Dev</span>
+              <span class="dot"></span>
 
-        </li>
-        <li>
-          <img src="image/Meduim.png" alt="" />
+              <span>2015</span>
+            </div>
+            <div class="description">
+              A daily selection of privately personalized reads; no accounts
+              or sign-ups required.
+            </div>
+            <div class="button-links">
+              <ul>
+                <li>html</li>
+                <li>css</li>
+                <li>javascript</li>
+              </ul>
+            </div>
+            <a href="#" class="project">See Project</a>
```

Comment:
- [ ] Kindly according to the requirements add here all the button properties for the hover, pressed, and disabled as shown in Figma and because it is very good for user experience (UX)✅✅💯💯